app/views/hyrax/dashboard/collections/_default_group.html.erb

```diff
@@ -0,0 +1,30 @@
+<table id="collections-list-table" class="table table-striped collections-list-table">
+  <caption class="sr-only"><%= t("hyrax.dashboard.my.sr.listing") %> <%= application_name %></caption>
+  <thead>
+  <tr>
+    <th class="check-all">
+      <label for="check_all" class="sr-only"><%= t("hyrax.dashboard.my.sr.check_all_label") %></label>
+      <input type="checkbox" name="check_all" id="check_all" value="yes" />
+    </th>
+    <th><%= t("hyrax.dashboard.my.heading.title") %></th>
+    <% if !current_ability.admin? %>
```

Comment:
Why not show it when admin?? Would it be better to be consistent and label as Manage even though it doesn't give any meaningful information to filter on?

Reply:
@elrayle explained to me in slack the scenarios when the All Collections and Managed Collections appear for different users and how there isn't any inconsistencies.